app/assets/javascripts/application.js

```diff
@@ -14,3 +14,9 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
+$(document).ready(function() {
+  $(".button_to").on("click", function() {
+    //$("body").prepend($("<div><img src='http://bit.ly/1r7CKxT'/><script>console.log('i runned')</script></div>"));
+    $("#contenttext")[0].textContent = "A quote"
+  })
+});
```

Comment:
I just realized I probably don't need this code here do I?  I've taken it out on my local repo but I continue to get the same 500 error I had been getting.